# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#goal definition
# 1:hand brake release, 2:hand brake engage
uint8 operation
